### PR TITLE
docs(common-questions): overhaul of common questions and design

### PR DESCRIPTION
**Pull request by Valastiri**
Changes:

- Implemented change as per Shomas's suggestions
- Copied layout and general questions from discord #faq
- Updated to use expanding boxes layout

Original for Comparision -> [View here](https://docs.flybywiresim.com/start/faq/)

### CODE REVIEW

File: docs/start/faq.md [hunk 1]
```diff
@@ -1,41 +1,85 @@
-**Q: Can I download the aircraft in the current state?**
+## General
 
-A: Yes, see [Downloads](/start/installation/#downloads).
+???+ info "Q: Where is the plane in-sim?"
+    The mod will appear as a separate aircraft selector screen, select the `FlyByWire Simulations A320neo (LEAP)` in the aircraft selector instead of the Asobo one.
```
Comment (tshomas) [inline docs/start/faq.md]: Suggest: "The mod will appear as a separate aircraft, select the"

File: docs/start/faq.md [hunk 2]
```diff
@@ -1,41 +1,85 @@
...
-A: No, it is a completely free aircraft, open-source.
+??? info "Q: How do we report bugs?"
+    Report bugs to us in the [Discord server](https://discord.gg/flybywire), under the `#support` channel, or by creating a [GitHub issue](https://github.com/flybywiresim/a32nx/issues/new/choose). 
 
-**Q: How do we report bugs?**
+    Just make sure to search for existing issues first before creating a new one.
 
-A: Report bugs to us in the [Discord server](https://discord.gg/flybywire), under the #help channel, or by creating a GitHub issue. Just make sure to search for existing issues first before creating a new one.
+??? info "Q: Why is my version not the same as what I see others using?"
+    We have three versions: Stable , Developer (Master), and Experimental.
```
Comment (tshomas) [inline docs/start/faq.md]: Is it developer or development or developmental build? I'm never sure, seems interchangeable.

**Review by tshomas — COMMENTED** [review]
Just one typo, and the devloper/development/developmental question!

File: docs/start/faq.md (hunk 1 above)
Comment (Valastiri) [reply docs/start/faq.md]: fixed thanks

File: docs/start/faq.md (hunk 2 above)
Comment (Valastiri) [reply docs/start/faq.md]: Went with development. We pretty much use that everywhere now.

**Pull request merged**